### PR TITLE
Workflow fixes

### PR DESCRIPTION
**Pull request by RemindD**
- Change k8s Activation Custom resource

   - stagehistory -> stageHistory
   - remove unneeded errorMessage field

- Change activation status to Paused if latest stage status is paused
- Change redis state provider separator to "*" since ":" can be used by reference

### CODE REVIEW

File: coa/pkg/apis/v1alpha2/providers/states/redisstate/redisstate.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ var rLog = logger.NewLogger("coa.runtime")
 
 const (
 	entryCountPerList = 100
-	separator         = ":"
+	separator         = "*"
```
Comment (LeiQL) [inline coa/pkg/apis/v1alpha2/providers/states/redisstate/redisstate.go:32]: Can be constant value. Or this is internal used only?

File: k8s/apis/workflow/v1/activation_types.go [hunk 2]
```diff
@@ -17,10 +17,9 @@ import (
 type ActivationStatus struct {
 	Status               v1alpha2.State `json:"status,omitempty"`
 	StatusMessage        string         `json:"statusMessage,omitempty"`
-	ErrorMessage         string         `json:"errorMessage,omitempty"`
 	ActivationGeneration string         `json:"activationGeneration,omitempty"`
 	UpdateTime           string         `json:"updateTime,omitempty"`
-	StageHistory         []StageStatus  `json:"stagehistory,omitempty"`
+	StageHistory         []StageStatus  `json:"stageHistory,omitempty"`
```
Comment (msftcoderdjw) [inline k8s/apis/workflow/v1/activation_types.go:22]: Please inform @FireDefend about this change.

File: k8s/apis/workflow/v1/activation_types.go [hunk 3]
```diff
@@ -17,10 +17,9 @@ import (
 type ActivationStatus struct {
 	Status               v1alpha2.State `json:"status,omitempty"`
 	StatusMessage        string         `json:"statusMessage,omitempty"`
-	ErrorMessage         string         `json:"errorMessage,omitempty"`
```
Comment (LeiQL) [inline k8s/apis/workflow/v1/activation_types.go:20]: We don't need the error message anymore?

Comment (RemindD) [reply k8s/apis/workflow/v1/activation_types.go:20]: It's recorded per stage

File: k8s/apis/workflow/v1/activation_types.go (hunk 2 above)
Comment (RemindD) [reply k8s/apis/workflow/v1/activation_types.go:22]: Yes. Shawn already knew that, and the change is in the swagger change,

File: coa/pkg/apis/v1alpha2/providers/states/redisstate/redisstate.go (hunk 1 above)
Comment (lirenjie95) [reply coa/pkg/apis/v1alpha2/providers/states/redisstate/redisstate.go:32]: It's used to solve this issue: https://github.com/eclipse-symphony/symphony/issues/389

Comment (RemindD) [reply coa/pkg/apis/v1alpha2/providers/states/redisstate/redisstate.go:32]: This is only used in Redis state provider so I think define in this module is enough.

**Review by msftcoderdjw — APPROVED** [review]

**Comment by msftcoderdjw** [comment]
Merged this PR as higher priority since this affects the bugbash builds.

**Pull request merged**